src/utils.ts

```diff
@@ -36,7 +36,7 @@ export const fromHexString = (hexString: string): Uint8Array => {
 };
 
 export const toHexString = (bytes: Uint8Array) =>
-  bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
+  '0x' + bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
```

Comment:
I prefer you don't modify this function. Add 0x where it's relevant instead

Reply:
done